docs/available-filters/record-encryption/aws-kms/creating_keys.adoc

```diff
@@ -0,0 +1,27 @@
+:aws:  https://docs.aws.amazon.com/
+
+= AWS Key Management Service
+
+As the Administrator, use either the AWS Console or CLI to
```

Comment:
I've been logging on to the AWS Console as the Administrator, then using the CloudShell to run the commands.  I haven't mentioned CloudShell here as it seems a bit prescriptive.

Reply:
I'll leave it out.